tools/serverpod_cli/lib/src/analyzer/entities/definitions.dart

```diff
@@ -374,12 +374,14 @@ class ForeignRelationDefinition extends RelationDefinition {
     String? name,
     required this.parentTable,
     required this.foreignFieldName,
-    this.onDelete = onDeleteDefault,
+    this.onDelete = onUpdateDefault,
```

Comment:
Chore: Wrong field here right?

```suggestion
    this.onDelete = onDeleteDefault,
```